examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -14,18 +14,19 @@
 // limitations under the License.
 //
 
-pub mod proto;
+pub mod proto {
+    include!(concat!(env!("OUT_DIR"), "/oak.examples.abitest.rs"));
```

Comment:
I would not mind too much, if it wasn't for the fact that that's against [Rust best practices](https://doc.rust-lang.org/cargo/reference/build-scripts.html#outputs-of-the-build-script), and that's why we ended up with all the [logic to second-guess whether things need to be rebuilt or not](https://github.com/project-oak/oak/blob/9bfc2a4a5bb9d7f77fbb114d7210fdf284b892aa/sdk/rust/oak_utils/src/lib.rs#L70-L172) (which is now gone).

But I agree that discoverability of the API is not great now, we should think about that. If nothing else, we should be able to bring up a local `cargo doc` instance and explore through that.

Reply:
Yeah, it would be nice if tools could find the code (e.g. racer, which has had an [open issue](https://github.com/racer-rust/racer/issues/191) for supporting `include!`d code for 5 years now).  Neat idea to use `cargo doc` though.